src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link should reference a CSS file, not an SCSS file. Change `href="styles/index.scss"` to `href="styles/index.css"` to ensure the browser can properly load the styles.